resources/uk/gov/hmcts/infrastructure/check-deprecated-postgres.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+output=$(grep -R --include=*.tf "hmcts/cnp-module-postgres" | wc -l)
+
+if [ $(echo $output | tr -d " ") = 0 ]; then
+  echo "Not using cnp-module-postgres module, this is good"
```

Comment:
probably don't need to echo anything in this case

Reply:
It doesn't actually output anything. Initially I tried something like [this](https://github.com/hmcts/cnp-jenkins-library/blob/9d751c190da3bc9b6d8326afc0cacb3ad20577c5/resources/uk/gov/hmcts/helm/warn-about-aad-identity-preview-hack.sh#L6-L15) but it wasn't giving me consistent results for some reason